### DISCUSSION TITLE
Is it possible to load and display a PDF file in memory using WebView2?

### DISCUSSION THREAD

**Discussion by LITTOMA** [unoplatform/uno#14514] [Q&A]
Can WebView2 display a PDF file loaded directly from memory? Looking for guidance or examples for this process. Any suggestions?

**Comment by jeromelaban** [comment]
Depending on the target, the only way to do this would be to use a [data urls](https://developer.mozilla.org/en-US/docs/web/http/basics_of_http/data_urls). Create one in C#, then you may be able to navigate to it, if the browser supports the file you're giving it.
